### ISSUE TITLE
bazel build error

### ISSUE THREAD

**Issue by 10212** [assyafii/ONOS-P4-INT#2]
Hello,have you ever met similar trouble?
![Screenshot from 2023-04-09 14-29-56](https://user-images.githubusercontent.com/30381236/230758208-ba924d87-5b55-460c-ae64-737bf4e1e068.png)

**Comment by assyafii** [comment]
Hello, have you changes & replaces http to https ?

https://luthfi.dev/posts/lab-p4-int-in-band-network-telemetry-with-onos-from-scratch/

You can see on step 4. Build onos

**Comment by 10212** [comment]
Yes,I did it follow your blog.But still occur error.

**Comment by 10212** [comment]
> 
sorry for bother.
I did it in wrong folder.Now I'm try it in correct way.
sorry again.
ROZ

**Comment by assyafii** [comment]
Nice @10212 good work, already success ?

**Comment by 10212** [comment]
> Nice @10212 good work, already success ?

Thank you very much. This is the most detailed blog I've come across in Try INT.
I've finished! And I got the same result as in your blog.But there's a little question, why is the queue occupancy always zero? I think that's kind of weird
![QQ截图20230417164059](https://user-images.githubusercontent.com/30381236/232683362-877c2c67-9427-4b7d-a172-8a7f27a3db09.png)
Thanks again, this blog is so helpful https://luthfi.dev/posts/lab-p4-int-in-band-network-telemetry-with-onos-from-scratch/
